website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,24 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
+            [
+                '/project/<pid>/osfstorage/<fid>/tags/<tag>/',
```

Comment:
@brianjgeiger Shouldn't this be `POST /project/<pid>/osfstorage/<fid>/` with the tag in the JSON request body?